sycl/source/detail/kernel_bundle_impl.hpp

```diff
@@ -504,6 +544,7 @@ class kernel_bundle_impl {
   // from any device image.
   SpecConstMapT MSpecConstValues;
   bool MIsInterop = false;
+  bundle_state MInitialState;
```

Comment:
Minor. Suggest renaming it to `MState` since AFAICS it correctly represents the current state at any time.

Reply:
It didn't with the old variant of `add_kernel` and I was not sure whether or not other places could bring images into other states, but it appears not to so I have renamed `MState` and simplified `get_bundle_state` to simply return it.